pandas/core/groupby/ops.py

```diff
@@ -392,6 +392,13 @@ def _call_cython_op(
                         values[mask] = True
             values = values.astype(bool, copy=False).view(np.int8)
             is_numeric = True
+        elif (
+            self.how in ["median", "sem", "std", "var"]
+            and "skipna" in kwargs
+            and not kwargs["skipna"]
+        ):
+            # if skipna=False we don't want to use masks created for Nullable dtypes
+            mask = None
```

Comment:
The masks indicate where NA values are, don't we need to use them?

Reply:
Correct, we had misunderstood how the masks were being used.